src/System.Composition.Convention/src/System/Composition/Convention/ConventionBuilder.cs

```diff
@@ -340,22 +340,22 @@ private List<Attribute> ReadParameterCustomAttributes(Type reflectedType, System
             return cachedAttributes;
         }
 
-        private static bool IsGenericDescendentOf(TypeInfo openType, TypeInfo baseType)
+        private static bool IsGenericDescendentOf(TypeInfo descendentType, TypeInfo baseType)
```

Comment:
I was debating for a while with the naming..

Basically I removed "openType" because the type could be open or not with this change. I totally agree with the name u suggested, I will rename it to derivedType when I get home